docs/setup/settings/options.md

```diff
@@ -38,3 +38,7 @@ Configure **Options** to set key default behaviors for the system. Click **`Edit
     Click on the `Save` button to apply the changes to your account.
 
     <img src="/setup/img/siprewrite.png">
+
+=== "Custom Pages"
+
+   Custom Pages allow you to add the created Apps to the **Dashboard** and the **Anyedge**.

```

Comment:
They are at the bottom on general in options and provide some info both separately dashboard and  AnyEdge Stats

Reply:
done
 Custom Pages allow you to add the created Pages from the IDE to the **Dashboard** and the **Anyedge**. It provides respective information on the Dashboard and Anyedege.